crates/primitives/src/compression/mod.rs

```diff
@@ -1,16 +1,26 @@
+#[cfg(feature = "std")]
 use std::{cell::RefCell, thread_local};
+#[cfg(not(feature = "std"))]
+use zstd::bulk::Decompressor;
+#[cfg(feature = "std")]
 use zstd::bulk::{Compressor, Decompressor};
```

Comment:
There is a lot of feature gating here. Instead of `use` on top of the file could we just use the full path.
For example instead of:
```
[cfg(feature = "std")]
 use zstd::bulk::{Compressor, Decompressor};
```
we could:
`zstd::bulk::Compressor` in line 26 *the same for other imports)